Client/mods/deathmatch/logic/luadefs/CLuaAudioDefs.cpp

```diff
@@ -1389,6 +1393,786 @@ int CLuaAudioDefs::GetSoundEffects(lua_State* luaVM)
+                            lua_setfield(luaVM, -2, EnumToString(I3DL2Reverb::REVERB_DELAY));
+
+                            lua_pushnumber(luaVM, fxI3DL2ReverbParams.flDiffusion);
+                            lua_setfield(luaVM, -2, EnumToString(I3DL2Reverb::DIFFUSION));
+
+                            lua_pushnumber(luaVM, fxI3DL2ReverbParams.flDensity);
+                            lua_setfield(luaVM, -2, EnumToString(I3DL2Reverb::DENSITY));
+
+                            lua_pushnumber(luaVM, fxI3DL2ReverbParams.flHFReference);
+                            lua_setfield(luaVM, -2, EnumToString(I3DL2Reverb::HF_REFERENCE));
+                            return 1;
+                        }
+                        break;
+                    }
+                    case BASS_FX_DX8_PARAMEQ:
+                    {
+                        BASS_DX8_PARAMEQ fxParameqParams;
+                        if (pSound->GetFxEffectParameters(eEffectType, &fxParameqParams))
+                        {
+                            lua_createtable(luaVM, 0, 3);
+
+                            lua_pushnumber(luaVM, fxParameqParams.fCenter);
+                            lua_setfield(luaVM, -2, EnumToString(ParamEq::CENTER));
+
+                            lua_pushnumber(luaVM, fxParameqParams.fBandwidth);
+                            lua_setfield(luaVM, -2, EnumToString(ParamEq::BANDWIDTH));
+
+                            lua_pushnumber(luaVM, fxParameqParams.fGain);
+                            lua_setfield(luaVM, -2, EnumToString(ParamEq::GAIN));
+                            return 1;
+                        }
+                        break;
+                    }
+                    case BASS_FX_DX8_REVERB:
+                    {
+                        BASS_DX8_REVERB fxReverbParams;
+                        if (pSound->GetFxEffectParameters(eEffectType, &fxReverbParams))
+                        {
+                            lua_createtable(luaVM, 0, 4);
+
+                            lua_pushnumber(luaVM, fxReverbParams.fInGain);
+                            lua_setfield(luaVM, -2, EnumToString(Reverb::IN_GAIN));
+
+                            lua_pushnumber(luaVM, fxReverbParams.fReverbMix);
+                            lua_setfield(luaVM, -2, EnumToString(Reverb::REVERB_MIX));
+
+                            lua_pushnumber(luaVM, fxReverbParams.fReverbTime);
+                            lua_setfield(luaVM, -2, EnumToString(Reverb::REVERB_TIME));
+
+                            lua_pushnumber(luaVM, fxReverbParams.fHighFreqRTRatio);
+                            lua_setfield(luaVM, -2, EnumToString(Reverb::HIGH_FREQ_RT_RATIO));
+                            return 1;
+                        }
+                        break;
+                    }
+                }
+            }
+        }
+        else if (pPlayer)
+        {
+            // TODO: Make it work with player voice
```

Comment:
It currently doesn't work with player voice. Should we leave that and make a new PR later to add support for player voice?

Reply:
Yes.